python/tests/hypothesis/arcticdb/test_resample.py

```diff
@@ -26,12 +29,23 @@ def date(draw, min_date, max_date):
 
 @st.composite
 def dataframe(draw):
-    index = hs_pd.indexes(elements=date(min_date=MIN_DATE, max_date=MAX_DATE).filter(lambda d: d is not pd.NaT), min_size=1)
-    columns = [hs_pd.column(name=f"col_{dtype}", dtype=dtype) for dtype in COLUMN_DTYPE]
+    '''
+    The dataframe returned will contain 3 64 bit columns with float64, int64, uint64 values.
+    However the range of values in them will be limited due to the fact that we will run sum, mean operations which have internal 
+    maximum of float64. Thus if we generate values to max 64 bits we would achieve overflow and this
+    could lead to failures in comparison to Pandas. Ints are limited to 62 bit and floats to 60 bits 
+    '''
+    index = hs_pd.indexes(elements=date(min_date=MIN_DATE, max_date=MAX_DATE)
+                          .filter(lambda d: d is not pd.NaT), min_size=1)
+    columns = [column(name="col_float", elements=st.floats(min_value=-1e18, max_value=1e18, 
+                                                           allow_nan=False, allow_infinity=False), dtype=np.float64),
+               column(name="col_int", elements=st.integers(-(2**61), 2**61 - 1), dtype=np.int64),
+               column(name="col_uint", elements=st.integers(0, 2**62 - 1), dtype=np.uint64)]
```

Comment:
64 for bit floats have 1 sign bit and 11 bits for exponent, they cannot represent 61 bit integers correctly. The largest int64 bit that can be represented without numerical errors is 2^53.